components/org.wso2.carbon.identity.api.user.authorized.apps/org.wso2.carbon.identity.rest.api.user.authorized.apps.v2/src/main/java/org/wso2/carbon/identity/rest/api/user/authorized/apps/v2/impl/ApplicationsApiServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+/*
+* Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package org.wso2.carbon.identity.rest.api.user.authorized.apps.v2.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.wso2.carbon.identity.rest.api.user.authorized.apps.v2.ApplicationsApiService;
+import org.wso2.carbon.identity.rest.api.user.authorized.apps.v2.core.AuthorizedAppsService;
+
+import javax.ws.rs.core.Response;
+
+/**
+ * Implementation of {@link ApplicationsApiService}
```

Comment:
Shall we add a `.` at the end?

Reply:
Addressed in [892099c](https://github.com/wso2/identity-api-user/pull/129/commits/892099c6c5888c3333cacd8be25ef6a8adccda93)